### DISCUSSION TITLE
@orrious Pool

### DISCUSSION THREAD

**Discussion by orrious** [tagyoureit/nodejs-poolController#503] [My Pool]
20K Gallons PebbleTec (Caribbean Blue ).  1x IntelliFlo VS Pump, 1x SuperFlo Pump . Spa spillover into pool. Waterfalls. Pentair IC-40 SWG. Zodiac Mx6 Elite pool sweep. Cartridge filter, MasterTemp 400 gas heater. Plumbed for Solar water system. 7 IntelliValves, flow meter on auto-fill, 250G propane tank with level sensor, 4 temperature sensors, Atlas ORP/PH.

I currently have a hybrid system that is a combination of njs-PC, HA, Node-Red, [pool-i2c-mega](https://github.com/orrious/pool-i2c-mega), [mqtt-i2c](https://github.com/orrious/mqtt-i2c).   

![Equipment_Pad](https://user-images.githubusercontent.com/9359564/175572838-c4427ceb-879f-47de-9d2d-ff9f8ee76277.jpg)
![Relay_enclosure](https://user-images.githubusercontent.com/9359564/175573262-d0c8c55b-09da-41cb-9d18-404f185badf8.jpg)
![High_Voltage_enclosure](https://user-images.githubusercontent.com/9359564/175573435-1ab7dcde-72e8-4266-a007-41cc88b75b95.jpg)
![Low_Voltage_Enclosure](https://user-images.githubusercontent.com/9359564/175573565-7f599664-3177-4d0f-ab81-54edfe9c63ce.jpg)
[Pool Plumbing - Pool cleaner.pdf](https://github.com/tagyoureit/nodejs-poolController/files/8977704/Pool.Plumbing.-.Pool.cleaner.pdf)

**Comment by johnny2678** [comment]
OH COME ON!!! Such a neat and ordered pad, I'm so jealous!

Very very nice.

Solar? Still under construction?

Do you happen to have a parts list for extending the filter drain with the ball valve?  I never knew I needed that but have to have it now.

**Reply by orrious** [reply]
I pre-plumbed for solar, that way I didn't have to cut into anything when I put it on.

The filter drain is metric style.  It has an O-ring and the threads aren't tapered.  I'm also into salt water aquariums and have had to find interesting ways of converting between the two.  Pretty sure this is a 1-1/2" male NPT to slip adapter.  If you just try taking the O-ring off of the plug and putting it on the threaded adapter, the O-ring is going to push right out.  Take a "rats tail" file (round) the same diameter as the O-ring and file a round grove all the way around the base of the thread where it meets the base.  File it down until the O-ring fits in the grove.  Then thread the adapter into the filter housing.  You should see the O-ring go into the filter housing now.
![IMG_4419](https://user-images.githubusercontent.com/9359564/175577685-d029ba1d-9431-4b04-8b38-e959445f05fd.jpg)
![IMG_4436](https://user-images.githubusercontent.com/9359564/175577687-284d2004-e1fd-46c8-b13b-708803666eb4.jpg)
![IMG_4437](https://user-images.githubusercontent.com/9359564/175577688-2103b02c-0cf7-445f-8473-713e0cca4e82.jpg)
![IMG_4438](https://user-images.githubusercontent.com/9359564/175577689-70ba5b80-c36e-463c-a470-d48df837dd76.jpg)

**Comment by rstrouse** [comment]
Great job!  I love how you painted labels for all the equipment on the side of the house! ;)

Very neat and tidy.  Weird that I find beauty in the pool equipment pad as much as the pool itself these days.  My transition to full on geekdom is nearly complete.

**Comment by tagyoureit** [comment]
Yes, very nice!  I also love the ball valve idea.  My filter drain is so impossible to get to.  It's on the bottom of the filter which makes it even worse.

Also, what's the spigot for coming out of the pump before the filter?  I have one, too, and it's always perplexed me.

**Reply by orrious** [reply]
The spigot is for pressurizing your system to find leaks, drain excess water, and prime pump more easily.  

All of the fitting on the pool shell are threaded and can be plugged / sealed.  The pool company I chose to build with, plumbed the pool incorrectly.  The valves were in the wrong order to be able to automate the pool effectively, so they paid me to replumb the entire equipment pad.  This was after the pool was filled.  So in the middle of January 2021 (water temp was 56F) I scuba-ed to bottom to pull the drain cover and plug the pool so I could cut all the pipes at the equipment pad.  Not an enjoyable experience to say the least.

Needless to say, once I replumbed everything I wanted to make sure nothing leaked.  Hooking your house hose up to the spigot will put 40lbs of pressure on the pipes.  More than they will ever see in operation.  Close it up and if it holds pressure for 24 hrs, you're good to go.

**Comment by rock-crusher** [comment]
Very nice work orrious ! I can see you are a craftsman and take pride in what you do (unlike the plumber that plumbed it originally)… Lol.
I notice a regulator on the gas line to your heater. The line looks small diameter wise, is this a high pressure line, you on propane, or ?

**Reply by orrious** [reply]
I am on propane and you're right the 3/8 copper line is high pressure going into the low pressure regulator.  There is no natural gas where I am in Florida and electric would take forever to heat the spa.  Propane was my only choice.

Speaking of propane and propane accessories, I'm looking for cost effective ignition systems for fire pit and fire bowls...

**Reply by Telemachus1977** [reply]
When you find the cost effective ignition please let us know.   Still use a 1.5v AA ignitor and turn the gas on by hand. 

With summer heat I can easily heat my hottub up to 102 with Solar.  By the nighttime when we get in it is still upper 90s.

**Comment by Exit2Studios** [comment]
Nice pad!  Where's the pic(s) of the actual pool?? :-)

**Comment by truvec** [comment]
@orrious Nice Work! I see you are in FL. Do you know the paver manufacture for your deck? Im looking for that type and most places seem to have lots of charcoal in the color mix. Thanks
